components/brave_wallet/browser/fil_transaction.cc

```diff
@@ -183,6 +183,11 @@ absl::optional<std::string> FilTransaction::GetMessageToSign() const {
   value.SetIntKey("Method", 0);
   value.SetIntKey("Version", 0);
   value.SetStringKey("Params", "");
+  const std::string* nonce_value = value.FindStringKey("Nonce");
+  bool nonce_empty = nonce_value && nonce_value->empty();
+  if (nonce_empty) {
```

Comment:
Can you add a comment here saying something similar to what's documented on the `MPoolGetNonce` API? It says there, `if this is the first time wallet, this call will return an error. If the error contains "address not found", we assume the Nonce is 0, since no on-chain transactions reference this address.` 

I almost forgot that this is the way that API worked so thinking it would be useful for me if I come across this again later and forget.

Reply:
done